package.json

```diff
@@ -11,7 +11,7 @@
     "docs:api:build": "tsx ./scripts/buildApiDocs/index.ts",
     "docs:api:buildX": "tsx ./docs/scripts/api/buildApi.ts",
     "docs:link-check": "tsx ./docs/scripts/reportBrokenLinks.js",
-    "docs:build": "pnpm --filter docs build",
+    "docs:build": "pnpm run release:build && pnpm --filter docs build",
```

Comment:
This change means that it shows the watermark now in the docs: 

<img width="696" alt="SCR-20250414-bvxp" src="https://github.com/user-attachments/assets/92062791-84f7-4457-b83e-871a7467e831" />

https://deploy-preview-17214--material-ui-x.netlify.app/x/react-data-grid/#pro-version

Before, we would set `MTU5NjMxOTIwMDAwMA==` for the release info, now it gets generated at each docs deploy.

It also seems that it takes more time now to deploy Netlify: https://app.netlify.com/sites/material-ui-x/deploys?page=1&filter=master.

Do we really need to do this change? What's wrong with? 

```suggestion
    "docs:build": "pnpm --filter docs build",
```

Reply:
> Before, we would set MTU5NjMxOTIwMDAwMA== for the release info, now it gets generated at each docs deploy.

IIUC, we're setting the package release timestamp to an older value to make it work with the timestamp of the (now expired) license stored in the `NEXT_PUBLIC_MUI_LICENSE` secret? Should we instead be updating that secret?

> Do we really need to do this change

The prod build of the docs now use the modules as they are published on NPM rather than path aliases. It allows us to catch build/export issues like these two, that we caught thanks to this PR:
https://mui-org.slack.com/archives/C042YB5RB3N/p1742407588778789
https://mui-org.slack.com/archives/C05SBSU8NPR/p1741040016736319

Reply:
> Should we instead be updating that secret?

It's another solution that we could use. 

> The prod build of the docs now use the modules as they are published on NPM rather than path aliases. It allows us to catch build/export issues like these two, that we caught thanks to this PR:

So, the question is then, is this worth the CI slowdown? I personally, I'm more the advoccat of: "everything instant, and I will fix bug quickly as they get reported in issues" 😄

Reply:
> Before, we would set MTU5NjMxOTIwMDAwMA== for the release info, now it gets generated at each docs deploy.

Can we keep doing that for the packages built specifically for the docs?